lib/components/viewers/pattern-row.js

```diff
@@ -123,91 +130,10 @@ export default class PatternRow extends Component {
     )
   }
 
-  _renderScheduleView = () => {
-    const {
-      pattern,
-      route,
-      stopTimes,
-      homeTimezone,
-      stopViewerArriving,
-      stopViewerConfig,
-      timeFormat
-    } = this.props
-    // sort stop times by next departure
-    let sortedStopTimes = []
-    const hasStopTimes = stopTimes && stopTimes.length > 0
-    if (hasStopTimes) {
-      sortedStopTimes = stopTimes
-        .concat()
-        .sort((a, b) => {
-          const aTime = a.serviceDay + a.scheduledDeparture
-          const bTime = b.serviceDay + b.scheduledDeparture
-          return aTime - bTime
-        })
-        // We request only x departures per pattern, but the patterns are merged
-        // according to shared headsigns, so we need to slice the stop times
-        // here as well to ensure only x times are shown per route/headsign combo.
-        // This is applied after the sort, so we're keeping the soonest departures.
-        .slice(0, stopViewerConfig.numberOfDepartures)
-    } else {
-      // Do not include pattern row if it has no stop times.
-      return null
-    }
-    const routeName = route.shortName ? route.shortName : route.longName
-    return (
-      <>
-        {/* header row */}
-        <div className='header'>
-          {/* route name */}
-          <div className='route-name'>
-            <b>{routeName}</b> To {pattern.headsign}
-          </div>
-        </div>
-        <div>
-          <div className='trip-table'>
-            {/* trips table header row */}
-            <div className='header'>
-              <div className='cell' />
-              <div className='cell'>BLOCK</div>
-              <div className='cell time-column'>DEPARTURE</div>
-            </div>
-
-            {/* list of upcoming trips */}
-            {hasStopTimes && (
-              sortedStopTimes.map((stopTime, i) => {
-                // Get formatted scheduled departure time.
-                const time = getFormattedStopTime(stopTime, homeTimezone, stopViewerArriving, timeFormat, true)
-                return (
-                  <div
-                    className='trip-row'
-                    style={{ display: 'table-row', marginTop: 6, fontSize: 14 }}
-                    key={i}>
-                    <div className='cell'>
-                      To {stopTime.headsign}
-                    </div>
-                    <div className='cell' style={{textAlign: 'center'}}>
-                      {stopTime.blockId}
-                    </div>
-                    <div className='cell time-column'>
-                      {time}
-                    </div>
-                  </div>
-                )
-              })
-            )}
-          </div>
-        </div>
-      </>
-    )
-  }
-
   render () {
     return (
       <div className='route-row'>
-        {this.props.showScheduleView
-          ? this._renderScheduleView()
-          : this._renderNextArrivalsView()
-        }
+        {this._renderNextArrivalsView()}
```

Comment:
Seems like the renderNextArrivalsView could be merged into the render function since it is always called only from here.

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/292/commits/bb270a7041700a19cc42eebf190a88fc04b0214c.